sklearn/preprocessing/data.py

```diff
@@ -397,6 +396,256 @@ def inverse_transform(self, X, copy=None):
         return X
 
 
+class RobustScaler(BaseEstimator, TransformerMixin):
+    """Standardize features by removing the median and scaling to IQR.
+
+    Centering and scaling happen independently on each feature (or each
+    sample, depending on the `axis` argument) by computing the relevant
```

Comment:
Maybe after this sentence one sentence "Interquantile range (IQR) is the range between 1st and 3rd quantile, that is between the 25th and 75th percentile" or something similar?